### DISCUSSION TITLE
Black screen Mario kart wii leaf cup, track ds desert hills

### DISCUSSION THREAD

**Discussion by Skybloomerskullfire** [Medard22/Dolphin-MMJR2-VBI#878] [Bug Report]
Too strange, but I think it is the capacity of the cell phone

https://github.com/user-attachments/assets/5f9f1a3c-16e5-4826-a369-29bc5a8146eb

**Comment by Medard22** [comment]
English please. Use Google translate if you have to. Also add other useful information like device information, Android version, backend, game settings, used hacks, etc.

**Reply by Skybloomerskullfire** [reply]
I understand what you are going through, it is a lot of pressure in the advancement of an emulator that is too powerful and complex, well, Happy New Year

**Reply by Medard22** [reply]
If you need help in debugging this issue, you really have to give me as much information as possible, otherwise I really can't help you.

Happy new year to you too.

**Reply by Skybloomerskullfire** [reply]
![Screenshot_20250103-223350](https://github.com/user-attachments/assets/2acd9bcf-7375-483f-9d5b-866e2bb30ff8)

**Comment by Skybloomerskullfire** [comment]
![Screenshot_20250103-223518](https://github.com/user-attachments/assets/a7a9e6fe-3b07-4596-bad2-16c78d5f878e)

**Comment by Medard22** [comment]
That is unfortunately pretty old device. First off check if you have a good dump of the game. If you have a PC you can use Dolphin to check the hash of the ISO/RVZ. Try to enable logging the Dolphin, play the game until you encounter this, and post the log here. It will be in the /mmmjr2-vbi/Logs/dolphin.log
